test/sigfd-deadlock.c

```diff
@@ -34,22 +35,32 @@ static int test_uring(int sfd)
 	struct io_uring ring;
 	int ret;
 
-	io_uring_queue_init(32, &ring, 0);
+	ret = io_uring_queue_init(32, &ring, 0);
+	if (ret)
+		return T_EXIT_FAIL;
 
 	sqe = io_uring_get_sqe(&ring);
 	io_uring_prep_poll_add(sqe, sfd, POLLIN);
-	io_uring_submit(&ring);
+	ret = io_uring_submit(&ring);
+	if (ret < 0) {
```

Comment:
This should be <= 0, as we expect a return of 1 here since we submitted one. Minor thing, won't happen, but if we're making changes...